ext/fetch/lib.rs

```diff
@@ -459,10 +460,47 @@ impl Resource for FetchCancelHandle {
 }
 
 pub struct FetchRequestBodyResource {
-  body: AsyncRefCell<mpsc::Sender<std::io::Result<bytes::Bytes>>>,
+  body: AsyncRefCell<mpsc::Sender<Option<bytes::Bytes>>>,
   cancel: CancelHandle,
 }
 
+pub struct ByteStream {
```

Comment:
Please add some description here. Consider moving this to `ext/fetch/byte_stream.rs`, ideally with a unit test.